src/main/scala/inox/solvers/SolverFactory.scala

```diff
@@ -90,15 +90,27 @@ object SolverFactory {
   // extract <exec> in "smt-z3:<exec>"
   private def getZ3Executable(name: String): String = name.drop(7)
 
-  def getFromName(name: String, force: Boolean = false)
+  def getFromName(noIncName: String, force: Boolean = false)
```

Comment:
I don't think it's very clean to change the API to include the no-inc detail. (I mean, I think keeping `name` is cleaner even though it means making more changes in the method body.)